### DISCUSSION TITLE
Some files were not synced properly ?

### DISCUSSION THREAD

**Discussion by jeromebaur** [GetPublii/Publii#1526] [Help] [answered]
Hello dear all, please I need some help. Several time Publii blocked the sync and the message is : ”Some files were not synced properly - please check the hard upload-errors-log.text file using the Tools Log Viewer tool”. And in that Tool I find hundreds of : ”ERROR UPLOAD FILE: httpdocs/fr/media/posts/96/responsive/EKO-cover-copie-sm.JPG
[Thu, 30 May 2024 11:03:26 GMT] Error: Client is closed because read ECONNRESET (data socket)” - And for many files. Unfortunately I'm not good to manage the code and my web site is blocked again. First time I had to use FileZilla, backup all and reload everything, but it's the hard way. Did someone have an idea to fix it ? 10000 thanks !

**Comment by dziudek** [comment, marked as answer]
@jeromebaur - if your website is big, maybe it is a good approach to use manual deployment for the first time, and then continue with other sync methods. I am working on few improvements to the sync methods so in the future releases of Publii bigger websites shouldn't be a problematic to upload

**Reply by jeromebaur** [reply]
Hello, sorry for the delay, since your last message I followed your instructions. I rebooted and I cleaned the server of my website to make a manual deployment. Then, the sync was working again until the same problem came back. So, I made all this process again. Now the sync is working but I'm not really zen :) I look forward the next update and I hope it will fix it. Thank you Jane a nice day !
